docs/content/using/using_hostdefinition.md

```diff
@@ -0,0 +1,72 @@
+# Using dynamic host definition
+
+Dynamic host connectivity removes the necessity for manual host definitions. This is also facilitates the scaling out process of a node. 
+
+The `HostDefiner` identifies the hosts available for host definition on each storage system and controls each of the host definitions. To see the statuses of the managed host definitions by the host definer, use the `hostdefinition` command.
+
+Use the `hostdefinitions` command to see the phase status of all definitions on the storage side.
+
+     $> kubectl get hostdefinitions
+
+|Phase|Description|
+|---------|--------|
+|Ready|Host definition is created and is working as expected.|
+|PendingForCreation|Host definition did not complete during the last attempt. The `HostDefiner` will try again.|
+|PendingForDeletion|Host deletion did not complete during the last attempt. The `HostDefiner` will try again.|
+|Error|Host definition or deletion did not complete and will not try again.|
+
+## Recovering from an Error state
+
+If any of the host definitions have an Error status, follow this procedure to have the `HostDefiner` reattempt to define the hosts.
+
+1. Undeploy the CSI node pod from the relevant node that the `HostDefinition` is a part of.
```

Comment:
should we elaborate about how (`IBMBlockCSI` node affinity)?

Reply:
@matancarmeli7 ?
I'm not sure what this means

Reply:
@oriyarde I think we can suggest they use it, but I don't think we need to give them examples, because this is another thing that we need to maintain and there are some ways to do it, what do you think?

Reply:
I agree this is not part of the host definer docs